packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -4,6 +4,7 @@
   "force_auth_web_login_authorize_org_text": "SFDX: Authorize an Org",
   "force_auth_access_token_authorize_org_text": "SFDX: Authorize an Org using Session ID",
   "force_auth_logout_all_text": "SFDX: Log Out from All Authorized Orgs",
+  "force_auth_logout_default_text": "SFDX: Log Out from Default Org",
```

Comment:
@sbudhirajadoc Display title for command as it appears in the command palette.

Reply:
Looks good!